addons/web/controllers/main.py

```diff
@@ -1064,7 +1053,7 @@ def _render_template(self, **d):
             monodb = db_monodb()
             if monodb:
                 d['databases'] = [monodb]
-        return Markup(env.get_template("database_manager.html").render(d))
+        return request.render('web.database_manager', d)
```

Comment:
How are we going to render the database manager without any database now?

Reply:
I've made a commit to have that case covered. I'll squash it if you validate.
https://github.com/odoo-dev/odoo/commit/d9218ce4fb59769058d0ea9da28b45980ed6504c